docs/README.md

```diff
@@ -9,489 +10,579 @@
 
 ### Variables
 
-* [KECCAK256_NULL](#keccak256_null)
-* [KECCAK256_NULL_S](#keccak256_null_s)
-* [KECCAK256_RLP](#keccak256_rlp)
-* [KECCAK256_RLP_ARRAY](#keccak256_rlp_array)
-* [KECCAK256_RLP_ARRAY_S](#keccak256_rlp_array_s)
-* [KECCAK256_RLP_S](#keccak256_rlp_s)
-* [MAX_INTEGER](#max_integer)
-* [TWO_POW256](#two_pow256)
-* [publicToAddress](#publictoaddress)
-* [setLength](#setlength)
-* [stripZeros](#stripzeros)
+* [BN](README.md#bn)
+* [Buffer](README.md#const-buffer)
+* [KECCAK256_NULL](README.md#const-keccak256_null)
+* [KECCAK256_NULL_S](README.md#const-keccak256_null_s)
+* [KECCAK256_RLP](README.md#const-keccak256_rlp)
+* [KECCAK256_RLP_ARRAY](README.md#const-keccak256_rlp_array)
+* [KECCAK256_RLP_ARRAY_S](README.md#const-keccak256_rlp_array_s)
+* [KECCAK256_RLP_S](README.md#const-keccak256_rlp_s)
+* [MAX_INTEGER](README.md#const-max_integer)
+* [TWO_POW256](README.md#const-two_pow256)
+* [assert](README.md#const-assert)
```

Comment:
This is the case I mentioned in the issue, having this `assert` here is completely nonsense.